### PR TITLE
altered destroy link to fix bug

### PR DESCRIPTION
**Pull request by meghart**
Fixed the link. It was a simple thing. Tested it this time too.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     rails_blogger_engine (0.0.2)
-      rails (~> 4.2.1)
+      rails (~> 4.2, >= 4.2.1)
```
Comment (meghart) [inline Gemfile.lock]: This is so any version of rails 4.2.~ works without having to upgrade.

File: app/controllers/rails_blogger_engine/articles_controller.rb [hunk 2]
```diff
@@ -45,10 +45,11 @@ def update
 
     # DELETE /articles/1
     def destroy
-      @article = Article.find(params[:id])
-      @article.destroy
-      redirect_to articles_url, notice: 'Article was successfully destroyed.'
+    @article.destroy
+    respond_to do |format|
```
Comment (meghart) [inline app/controllers/rails_blogger_engine/articles_controller.rb]: I rewrote this section, but I don't know that you necessarily have to use it.

File: app/views/rails_blogger_engine/articles/show.html.erb [hunk 3]
```diff
@@ -9,7 +9,7 @@
     Submitted <%= time_ago_in_words(@article.created_at) %>
     <%= link_to 'Edit', edit_article_path(@article) %> |
     <%= link_to 'Back', articles_path %> |
-    <%= button_to 'Destroy', articles_path(@article), method: :delete, data: { confirm: 'Are you sure?' } %>
+    <%= button_to 'Destroy', @article, method: :delete, data: { confirm: 'Are you sure?' } %>
```
Comment (meghart) [inline app/views/rails_blogger_engine/articles/show.html.erb]: This is where the error was happening. I simplified the articles_path(@article) to just @article and now it works.

**Comment by erickuhn19** [comment]
When I pulled it down locally I was getting some database error, so I just used the simplification in show.html.erb and it worked. I will close this request but I used the change.

**Pull request closed without merging**